tools/pd-ctl/pdctl/command/hot_command.go

```diff
@@ -123,3 +155,69 @@ func parseOptionalArgs(cmd *cobra.Command, prefix string, args []string) (string
 	}
 	return prefix, nil
 }
+
+func parseHotRegionsHistoryArgs(args []string) (map[string]interface{}, error) {
+	startTime, err := strconv.ParseInt(args[0], 10, 64)
+	if err != nil {
+		return nil, errors.Errorf("start_time should be a number,but got %s", args[0])
+	}
+	endTime, err := strconv.ParseInt(args[1], 10, 64)
+	if err != nil {
+		return nil, errors.Errorf("end_time should be a number,but got %s", args[1])
```

Comment:
```suggestion
		return nil, errors.Errorf("end_time should be a timestamp,but got %s", args[1])
```

Reply:
the error generate because of converting to number,not timestamp